packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/UpdatePalletFrozenStatus.tsx

```diff
@@ -31,14 +31,14 @@ export const UpdatePalletFrozenStatus = () => {
       </Row>
       <Row>
         <InlineToggleWrap>
-          <Label>CRT feature</Label>
+          <Label>Creator Tokens</Label>
           <Tooltip tooltipText="You have the flexibility to enable or disable CRT feature.">
             <TooltipDefault />
           </Tooltip>
         </InlineToggleWrap>
         <ToggleCheckbox
```

Comment:
Again the goal is to match the design so what happens if you move `ToggleCheckbox` inside `InlineToggleWrap`:
```suggestion
          <ToggleCheckbox
            falseLabel="Disable"
            trueLabel="Enable" 
            name="updatePalletFrozenStatus.freeze"
            id="crt-feature-select"
          />
          <Tooltip tooltipText="You have the flexibility to enable or disable CRT feature.">
            <TooltipDefault />
          </Tooltip>
        </InlineToggleWrap>
```